### ISSUE TITLE
Script for uploading dialog flow

### ISSUE THREAD

**Issue by hyzhak** [internetarchive/internet-archive-google-action#33]
# The Goal 
We should be able automatically to populate the DialogFlow entity lists, which are related to our subject area. 

For example:
- [ ] categories like `concerts`, `78`: `collection names => collectionId`
- [ ] sub-categories like band names (creators): `creator name => collectionId`
- [ ] albums and concerts `album name => album name`
- [ ] genres: `genre name => genre name`
- [ ] songs titles: `title => title`
and etc

So we should match differently speaking titles to the "identifiers" (for example: `live, Concerts, Live Concerts) -> etree`, `78,  -> georgeblood`). Which we could use for https://web.archive.org/advancedsearch.php request.

# Pipeline
- [ ] it should be runnable through `npm run upload:<name>`
- [ ] all data should automatically download from IA and upload to Dialog Flow
- [ ] don't put tokens into the repository they should be got from `process.env`

# PS
The last but not least. We should know what is the limit of Dialog Flow? And how many data could we store here?

# Dependency
related #35

**Comment by bimlendra** [comment]
Done
